package.json

```diff
@@ -36,7 +36,8 @@
     "lint:prettier": "prettier --check \"{src,__tests__}/**/*.(js|ts|tsx)\"",
     "lint:eslint": "eslint ./src --ext .ts,.tsx",
     "fix:eslint": "yarn lint:eslint --fix",
-    "fix:prettier": "prettier . --write"
+    "fix:prettier": "prettier . --write",
+    "release": "./scripts/release.sh"
```

Comment:
awesome!
can this be prepublish? https://yarnpkg.com/package/prepublish
not sure there’s post publish, but slack notifier could go there if exist.

Reply:
not sure why we'd use a deprecated package (or why you'd link to yarn instead of npm) but yes, this should be a `postpublish` script if it's meant to run after a manual publish.

however, if this is meant to replace *publishing*, then we probably want to add a `prepublishOnly` script that files if the right env var isn't set, and tells the user to run `yarn release`.

Reply:
Because I can't `cd` into a directory within a script, I won't be able to update this to `prepublishOnly`